.kodiak.toml

```diff
@@ -4,12 +4,23 @@ version = 1
 [merge]
 
 method = "squash"
-
-notify_on_conflict = false
-
-dont_wait_on_status_checks = ["EIP Auto-Merge Bot", "Travis CI - Pull Request"]
-
+delete_branch_on_merge = true
 require_automerge_label = false
 
 # important
 block_on_neutral_required_check_runs = true
+
+# Add useful information & co-authors to the PR commit body
+[merge.message]
+include_coauthors = true
+include_pull_request_author = true
+include_pull_request_url = true
+
+# Keep dependencies up-to-date automatically 
+[merge.automerge_dependencies]
+versions = ["minor", "patch"] # do not automerge major version upgrades
+usernames = ["dependabot"]
```

Comment:
Does this bypass the need for an editor to approve? I dislike that from a supply chain attack perspective.

Reply:
Yes it does. I'll remove it then.